handlers/writer.go

```diff
@@ -37,12 +37,17 @@ func WriteExcelFile(w http.ResponseWriter, r *http.Request) {
 
 	file.WriteSheets(body.Sheets)
 
+	var filename = fmt.Sprintf("%s", time.Now().Format("2006-01-02_15:04:05"))
+	if body.Filename != "" {
+		filename = body.Filename
+	}
+
 	w.WriteHeader(http.StatusCreated)
 	w.Header().Set("Content-Type", "application/octet-stream")
 	w.Header().Set(
 		"Content-Disposition", fmt.Sprintf(
-			"attachment; filename=file_generated_%s.xlsx",
-			time.Now().Format("2006-01-02_15:04:05"),
+			"attachment; filename=%s.xlsx",
+			filename,
```

Comment:
Why ? This filename is only for testing purposes, the API will only return a octet stream, no filename is needed. The user needs to convert this octet stream into a correct XLSX file with his own techno (cURL, JS, or whatever)

Reply:
Yes you're right .... my bad. 

I close the PR